.github/workflows/sub-deploy-code-slot.yml

```diff
@@ -112,7 +115,7 @@ jobs:
           -n ${{ inputs.apiName }} \
           --src ./${{ inputs.apiName }}.zip \
           --slotName ${{ inputs.slotName }} \
-          --settings "INFO_SHA=${{ github.sha }}"
+          --settings "INFO_SHA=${{ github.sha }} MOCK_AUTH=${{ inputs.mockAuth }}"
```

Comment:
I'm not sure we need this here. We read from React env on the frontend, I don't think we would ever access this setting